README.md

```diff
@@ -909,6 +927,32 @@ path, err = versions.CreatePath("subject/repo/pkg/version")
 btManager.MavenCentralContentSync(params, path)
 ```
 
+#### Using ContentReader
+
+Some APIs return a content.ContentReader which allow reading the API output.
+Here's an example for how it can be used:
+
+```go
+reader, err := servicesManager.SearchFiles(searchParams)
+if err != nil {
+    return err
+}
+defer reader.Close()
+for currentResult := new(ResultItem); reader.NextRecord(currentResult) == nil; currentResult = new(ResultItem)  {
```

Comment:
I suggest adding the following comment above the loop:

// Iterate over the results